.github/workflows/run-end-to-end.yml

```diff
@@ -117,7 +117,7 @@ jobs:
             "rack",
             "uds-sinatra",
             *[f"sinatra{v}" for v in ["14", "20", "21", "22", "30", "31", "32", "40"]],
-            *[f"rails{v}" for v in ["32", "40", "41", "42", "50", "51", "52", "60", "61", "70", "71"]]
+            *[f"rails{v}" for v in ["50", "51", "52", "60", "61", "70", "71"]]
```

Comment:
I'll add in a later PR a job that check those variant on your 1.x branch to prevent any breakage on your CI